---
title: "&#39; Exit Try &#39; può trovarsi solo all'interno di un &#39; Provare a &#39; istruzione"
ms.date: 07/20/2015
ms.prod: .net
ms.technology: devlang-visual-basic
ms.topic: article
f1_keywords:
- vbc30393
- bc30393
helpviewer_keywords: BC30393
ms.assetid: b8651df3-a32f-478c-a6d8-aa0ef584155f
caps.latest.revision: "7"
author: dotnet-bot
ms.author: dotnetcontent
ms.openlocfilehash: 0eaf43bf42d1a323221d79dabf2bff531abcfe6f
ms.sourcegitcommit: 4f3fef493080a43e70e951223894768d36ce430a
ms.translationtype: MT
ms.contentlocale: it-IT
ms.lasthandoff: 11/21/2017
---
# <a name="39exit-try39-can-only-appear-inside-a-39try39-statement"></a>&#39; Exit Try &#39; può trovarsi solo all'interno di un &#39; Provare a &#39; istruzione
`Exit Try` può trovarsi solo all'interno di un'istruzione del blocco `Try` . È presente un'istruzione `Exit Try` ridondante oppure l'istruzione `Exit Try` si trova al di fuori dei limiti del blocco `Try` corrispondente.  
  
 **ID errore:** BC30393  
  
## <a name="to-correct-this-error"></a>Per correggere l'errore  
  
1.  Individuare e rimuovere l'istruzione `Exit Try` non necessaria.  
  
2.  Spostare il `Exit Try` istruzione nella posizione appropriata all'interno del codice.  
  
## <a name="see-also"></a>Vedere anche  
 [Istruzione Try...Catch...Finally](../../visual-basic/language-reference/statements/try-catch-finally-statement.md)  
 [Gestione delle eccezioni Panoramica per Visual Basic](http://msdn.microsoft.com/en-us/bb81af80-a735-4873-9711-6151a48e418a)
